app/controllers/solidus_paypal_commerce_platform/orders_controller.rb

```diff
@@ -10,7 +10,7 @@ def create
       authorize! :create, ::Spree::Order
 
       @order = ::Spree::Order.create!(
-        user: try_spree_current_user,
+        user: ::Spree.solidus_gem_version > ::Gem::Version.new('3.2.x') ? spree_current_user : try_spree_current_user,
```

Comment:
`.x` is not a supported syntax for Gem::Version, and I think it's being interpreted as a prerelease version, this should probably be just `::Spree.solidus_gem_version >= Gem::Version.new('3.2')` (we also only need to add `::` to `Spree` because it's being defined inside the extension namespace for decorators).

Reply:
Got it, interesting find - I pulled the syntax from other extensions that will probably need to be updated as well later
Ex: [solidus_product_assembly](https://github.com/solidusio-contrib/solidus_product_assembly/blob/c55fb0f9a3572d51dc5289ffeb247a158bc26b1d/app/decorators/models/solidus_product_assembly/spree/stock/inventory_unit_builder_decorator.rb#L21)